src/docs/perf/rendering/index.md

```diff
@@ -33,7 +35,29 @@ see the following docs:
 * [Performance best practices][]
 * [Flutter performance profiling][]
 
+## Mobile-only advice
+
+Do you see noticeable jank on your mobile app, but only on
+the first run of an animation? If so, see
+[Reduce shader animation jank on mobile][].
+
+[Reduce shader animation jank on mobile]: /docs/perf/rendering/shader
```

Comment:
For future PRs: maybe it's nice to include some of Filip's articles here so the mobile section won't look very empty compared to the web section.